lib/eip721Subraph.ts

```diff
@@ -4,7 +4,7 @@ import { contractDirectory } from 'lib/contracts';
 export const HIDDEN_NFT_ADDRESSES = [
   contractDirectory.borrowTicket,
   contractDirectory.lendTicket,
-];
+].map((a) => a.toLowerCase());
```

Comment:
do we need to double check that the addresses being compared are similarly normalized?

Reply:
they come back from the subgraph similarly normalized yup